doc/chips/10.rst

```diff
@@ -250,6 +250,78 @@ block while the rules for Phase 2 may be applied to the second, without a
 constant check to the location of the ``.init()`` call which is the linchpin for
 syntax 1.
 
+Child's fields could depend on parent's fields
+**********************************************
+
+We also considered allowing child's fields to be initialized based on
+parent's fields. For example, the child could control where, in its Phase 1,
+the parent's Phase 1 is invoked, not necessarily as the last step.
+
+The following example shows that based on the second syntax above.
+It also shows the option of having the child control when the parent's Phase 2
+is invoked.
+
+.. code-block:: chapel
+
+   class Parent {
+     var D: domain(1);
+     proc init(myRange) {
+       D = {myRange};  // Phase 1
+     } finalize {
+       ... Phase 2 code ...
+     }
+   }
+
+   class Child {
+     var xD: domain(1);
+     proc init(myRange, extra) {
+       super.init(myRange);   // initializes Parent.D
+       xD = D.expand(extra);  // initialize xD based on Parent.D
+     } finalize {
+       ... Phase 2 in Child ...
+       super.finalize();      // executes Phase 2 in Parent
+     }
+   }
+
+This approach seems to require the syntax 2.
+We chose not to incorporate it because it makes less clear
+how initialization occurs.
```

Comment:
How about "This approach seems to require syntax 2" or "seems to require the second syntax"?  The way this is written now is not proper English.  The second sentence feels informal in a manner similar to the point raised above.